examples/chat/public/main.js

```diff
@@ -119,8 +115,7 @@ $(function() {
   // options.prepend - If the element should prepend
   //   all other messages (default = false)
   const addMessageElement = (el, options) => {
-    var $el = $(el);
-
+    const $el = $(el);
     // Setup default options
     if (!options) {
       options = {};
```

Comment:
I'd suggest moving this to a default function parameter value.

Reply:
I don't feel to strongly about this change though, because there's additional "default options logic" that can't be moved into the parameter description, since you may define `fade` and not `prepend`, and you still want `prepend` to default to the value below (following the logic in this code).